db/sql/sql_test.go

```diff
@@ -411,6 +412,57 @@ var _ = Describe("Sql", func() {
 				Expect(resSql).To(Equal(expectedSql))
 				Expect(param).To(Equal(expectedParam))
 			})
+			It("should create LIKE query with one parameter", func() {
+				search_value := search.NewSearchField("123")
+				filter := map[string]interface{}{"test_string": search_value}
+
+				res, err := AddFilterToQuery(testSchema, query, filter, false)
+
+				Expect(err).ToNot(HaveOccurred())
+				resSql, param, err := res.ToSql()
+				Expect(err).ToNot(HaveOccurred())
+				expectedQuery = expectedQuery.Where(Like{"`test_string`": "%123%"})
+				expectedSql, expectedParam, _ := expectedQuery.ToSql()
+				Expect(resSql).To(Equal(expectedSql))
+				Expect(param).To(Equal(expectedParam))
+			})
+			It("should create LIKE query with one parameter and escape special characters", func() {
+				search_value := search.NewSearchField("\\1%2_3_a\\b%c")
+				filter := map[string]interface{}{"test_string": search_value}
+
+				res, err := AddFilterToQuery(testSchema, query, filter, false)
+
+				Expect(err).ToNot(HaveOccurred())
+				resSql, param, err := res.ToSql()
+				Expect(err).ToNot(HaveOccurred())
+				expectedQuery = expectedQuery.Where(Like{"`test_string`": "%\\\\1\\%2\\_3\\_a\\\\b\\%c%"})
+				expectedSql, expectedParam, _ := expectedQuery.ToSql()
+				Expect(resSql).To(Equal(expectedSql))
+				Expect(param).To(Equal(expectedParam))
```

Comment:
Above block of code is virtually identical to the one in the preceding test case. You can factor this out into a function.